.github/workflows/stale.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Close Stale PRs
+      uses: actions/stale@v5
+      with:
+        days-before-issue-close: -1
```

Comment:
```suggestion
        days-before-close: -1
```

Reply:
Too narrow

Reply:
I thought we wanted to keep issues open and only mark them stale?

Reply:
yes and the same for prs, no?

Reply:
ah no let's close them, if they are no changes made after a certain time.

Reply:
ok 👍